packages/wallet/src/PersonalWallet/PersonalWallet.ts

```diff
@@ -707,7 +712,7 @@ export class PersonalWallet implements ObservableWallet {
     );
   }
 
-  async getPubDRepKey(): Promise<Ed25519PublicKeyHex> {
-    return this.keyAgent.derivePublicKey(keyManagementUtil.DREP_KEY_DERIVATION_PATH);
+  getPubDRepKey(): Ed25519PublicKeyHex {
```

Comment:
This should be `async`, otherwise it won't work with remoteApi

Reply:
done in [274e020](https://github.com/input-output-hk/cardano-js-sdk/pull/936/commits/274e0201bc9165a19d955ff31a8c41e6bebc0f59)